app/build.gradle

```diff
@@ -20,12 +20,18 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
         }
     }
+    compileOptions {
```

Comment:
Is it necessary to specify the java version here ?

Reply:
I dunno why that's there, my ide might've added it lol. I will remove it